### ISSUE TITLE
Umatrix blocking lacks unblock interface and is unintuitive

### ISSUE THREAD

**Issue by rainbowdesign** [gorhill/uMatrix#510]
There are some cases where Umatrix blocks a website and replaces the site with:

uMatrix has prevented the following page from loading:
*link
*go back button

However the interface shows no clear way to create an exception here.
And since there are often sites that are blocked while still are not dangerous it happens that the user want to unblock it.
Next to the go back button should be an exception button to create an exception and visit the website if i still want to access the website. This way the interface would be more intuitive.

**Comment by gorhill** [comment]
Duplicate of #203.

Just open the uMatrix popup and remove the block rule.

**Comment by TeKett** [comment]
necro but its important info people need which you cant find anywhere by using google(untill now hopefully)

this was so simple it makes we want to cry, and im gonna call the creator of umatrix a retard for doing it this way cus its idiotic and makes people uninstall before noticing. atleast i almost did untill i noticed how simple it was.

how to unblock a site if umatrix blocks it (just for the search for solution)

if a site or page gets blocked
![screenshot_50](https://cloud.githubusercontent.com/assets/26717389/25098335/694e133a-23a8-11e7-98e2-931fca9a41d3.png)
what you do is just on the same page you disable the matrix for the current scoop and then you can save it to perm if you whant
![screenshot_51](https://cloud.githubusercontent.com/assets/26717389/25098481/067f8c6a-23a9-11e7-9c81-4fd97d23796c.png)
then you can just restart and its all fine
![screenshot_52](https://cloud.githubusercontent.com/assets/26717389/25098499/12f02b6c-23a9-11e7-98ad-5414a6811a96.png)

**Comment by TriMoon** [comment]
@TeKett , LMAO !
Sorry but if you don't know how to use something you installed, why use it at all ?
The whole **point** of uMatrix is to block things, so the **first** reaction you should have had is to check the uMatrix setting for the page you're on when you get such a message.

That said,
Either please remove the addresses in your screenshot and remove the screenshot of the final page to prevent **not_asked_for** advertisement.
(Ads work for me like red flags for bulls, eg. makes me aggravated)
